app/bindings/api/v0/bindings/nudged.rb

```diff
@@ -0,0 +1,301 @@
+=begin
+#OpenStax Event Capture API
+
+#The Event Capture API for OpenStax.  Requests to this API should include `application/json` in the `Accept` header.  The desired API version is specified in the request URL, e.g. `[domain]/api/v1/events`. While the API does support a default version, that version will change over time and therefore should not be used in production code! 
+
+OpenAPI spec version: 0.1.0
+
+Generated by: https://github.com/swagger-api/swagger-codegen.git
+Swagger Codegen version: 2.4.9
+
+=end
+
+require 'date'
+
+module Api::V0::Bindings
+  class Nudged
```

Comment:
This `Nudged` is the "v1" nudged schema?  If so, what happens when there is a "v2" with different fields?  It'll have nowhere to live.  I think "v1" has to be in the class name or module namespace or something.

Reply:
@jpslav 

Nudged is in the v1 namespace..when i add the nudged swagger schema to the swagger classes, it's added like this

```
0> DatumSwaggerFinder.new.datum_classes 
=> [Ec::Nudged::V1::NudgedSwagger]
```